src/common/images/uploadImagesDialog.component.tsx

```diff
@@ -135,11 +135,22 @@ const UploadImagesDialog = (props: UploadImagesDialogProps) => {
   return (
     <DashboardModal
       open={open}
+      locale={{
+        strings: {
+          dropPasteFiles: 'Drop images here or %{browseFiles}',
+        },
+        pluralize: (n) => {
+          if (n === 1) {
+            return 0;
+          }
+          return 1;
+        },
```

Comment:
```suggestion
        pluralize: (n) => n == 1 ? 0 : 1
```
Probably more concise, but obviously works fine either way.

Reply:
I copied this directly from uppy (https://github.com/transloadit/uppy/blob/bb82326d0dd8f999bcb99deeeaa924250c41ffae/packages/%40uppy/locales/template.ts#L6). I'll add a comment in the code linking so where i got this code from.

Reply:
Strange no idea why decided to use so many lines, but yeah that's fine.